Makefile

```diff
@@ -243,6 +243,12 @@ pid = /tmp/stunnel.pid
 [redis]
 accept = 127.0.0.1:6390
 connect = 127.0.0.1:6379
+[redis_3]
```

Comment:
@sazzad16 where is redis_2?

Reply:
@gkorland This piece of code is from #2139 . I think the OP left redis_2 for the connection in 6380 port.